app/models/delivery.rb

```diff
@@ -12,6 +12,7 @@ class Delivery < ActiveRecord::Base
   before_save :update_offer_state
   before_destroy :push_back_offer_state
   after_save :send_updates, if: "offer.scheduled?" && :process_completed?
+  after_destroy {send_updates :delete unless Rails.env.test? }
```

Comment:
Can we make tests pass without needing Rails.env.test? I'm a little wary of code that isn't run during tests. Thanks